datamade/how-to
# Add documentation of remote practices, add work-practices/ directory

## Overview

This PR:

- Adds a new `work-practices/` directory and moves docs for project management, user research, and investment time into it.
- Adds new documentation for remote work.
- Updates the main README to reflect this new directory structure.

Handles #54, closes https://github.com/datamade/hr/issues/21.

## Testing Instructions

* Click the links in the top-level README and confirm they all resolve.
* Navigate around the docs in the `work-practices/` directory and confirm they look good, images render, and links resolve.
* Review the prose of the remote work documentation and confirm it's accurate and clear.
